ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -82,6 +79,7 @@ object SqlLedger {
       ledgerId: Option[LedgerId],
       timeProvider: TimeProvider,
       acs: InMemoryActiveContracts,
+      packages: InMemoryPackageStore,
```

Comment:
Not sure I understand. Why do we use `InMemoryPackageStore` for the SqlLedger?

Reply:
This PR is just WIP so far. First part was to move ownership of the package store to the `Ledger` instance. Next part will be copying the given `InMemoryPackageStore` (populated by the CLI arguments and scenario loader) to the PostgreSQL database on startup, and then serving package management calls from the database.

Reply:
Alright, I'll shut up now and wait for your signal :)